authui/src/authflowv2/botprotection/botProtectionStandalonePage.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller } from "@hotwired/stimulus";
+import {
+  dispatchBotProtectionWidgetEventRender,
+  dispatchBotProtectionWidgetEventUndoRender,
+} from "./botProtectionWidget";
+
+/**
+ * Controller for bot protection standalone page `verify_bot_protection.html`
+ *  - Manage widget rendering
+ *
+ * @listens bot-protection-widget:ready-for-render
+ * @fires bot-protection-widget:render
+ * @fires bot-protection-widget:undo-render
+ * Expected usage:
+ * - Add `data-controller="bot-protection-standalone-page"` to a `<div>` element
+ */
+export class BotProtectionStandalonePageController extends Controller {
+  connect() {
+    document.addEventListener("bot-protection-widget:ready-for-render", () =>
+      dispatchBotProtectionWidgetEventRender()
+    );
+  }
+
+  disconnect() {
+    document.removeEventListener("bot-protection-widget:ready-for-render", () =>
+      dispatchBotProtectionWidgetEventRender()
+    );
```

Comment:
This does not work. The argument to addEventListener and removeEventListener MUST BE a constant.